File: BankingService/src/main/java/com/bankibgapp/bankingservice/model/request/UtilityPaymentRequest.java
package com.bankibgapp.bankingservice.model.request;

import lombok.Data;

import java.math.BigDecimal;
import java.util.UUID;


@Data
public class UtilityPaymentRequest {
    private UUID providerId;
    private BigDecimal amount;
    private String referenceNumber;
    private String account;

}

File: BankingService/src/main/java/com/bankibgapp/bankingservice/request/UtilityPaymentRequest.kt
package com.bankibgapp.bankingservice.request

import java.math.BigDecimal
import java.util.*

data class UtilityPaymentRequest @JvmOverloads constructor(
    var providerId: UUID,
    var amount: BigDecimal,
    var referenceNumber: String,
    var account: String
)